### ISSUE TITLE
Index 196 is out of bounds for axis 0 with size 196

### ISSUE THREAD

**Issue by brunohlopez** [davidraythompson/istutor#3]
I am running through the Notebook examples and have run into an issue in Notebook 13 OE Cubes. In chunk 4 where we begin to do PCA and segmentation I run into an error. The code of block starts with importing sklearn.segmentation import slic

![Screen Shot 2022-02-08 at 10 20 10 AM](https://user-images.githubusercontent.com/65701128/153051104-cff2b44c-869c-4efc-8b1e-80fb7206968a.png)

It seems the labels variable in that same chunk only produces a value of 196 and subsequently the lbl variable only goes up to 196.

When you run this block of code 

`color_labels = np.zeros((rows,cols))
for r in range(rows):
    for c in range(cols):
        color_labels[r, c] = color_index[labels[r, c]]  `

you get the index out of bounds error. Was wondering if anyone has run into this before.

**Comment by davidraythompson** [comment]
Thanks for this report.  I don't get this error running with skimage version 0.18.1.  My own notebook shows some minor differences with the repository version, so I pushed it to make sure the repo is up to date.    

My only other suggestion is that this chunk of code is only used for visualization, so if you get stuck you can comment out the rest of the cell, and the rest of the exercise should still work provided that the segmentation result is valid.
